CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ if(NOT Poco_FOUND)
   if(WIN32)
     list(APPEND POCO_CXX_FLAGS "/wd4244 /wd4530 /wd4577")
   else()
-    list(APPEND extra_cmake_args "-DCMAKE_C_FLAGS=-Wno-shift-negative-value")
+    list(APPEND extra_cmake_args "-DCMAKE_C_FLAGS=${CMAKE_C_FLAGS} -Wno-shift-negative-value")
```

Comment:
@asobhy-qnx while backporting this change to Dashing this question was raised https://github.com/ros2/poco_vendor/pull/29/files#r527339063

@gbiggs: Is this meant to be CMAKE_CXX_FLAGS?

Reply:
Yes it would make more sense to use CMAKE_CXX_FLAGS, the reason I used CMAKE_C_FLAGS was because it was already there from previous commit and I wanted to append to the flags the option -fPIC so I appended to CMAKE_C_FLAGS.